package.json

```diff
@@ -28,7 +28,30 @@
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
         "lint": "eslint --ext=.js,.jsx .",
-        "fix": "npm run lint -- --fix"
+        "fix": "npm run lint -- --fix",
+        "lint:scss": "stylelint '**/*.scss'",
+        "lint:scss:fix": "stylelint '**/*.scss' --fix",
+        "prepare": "husky install",
+        "svgo": "svgo -r -f ./"
+    },
+    "lint-staged": {
+        "**/*.{js,jsx}": [
+            "eslint --fix",
+            "git add"
```

Comment:
Do we need a `git add` here? Ideally we should allow users to git add necessary files ?

Reply:
I tough we can add those files after fixing them. Better we can allow users to add necessary files.